Makefile.include

```diff
@@ -350,6 +350,10 @@ ifneq (, $(filter clean, $(MAKECMDGOALS)))
     all $(BASELIBS) $(USEPKG:%=$(RIOTPKG)/%/Makefile.include) $(RIOTBUILD_CONFIG_HEADER_C) pkg-prepare: clean
 endif
 
+ifneq (,$(filter flash, $(MAKECMDGOALS)))
```

Comment:
Wouldn't it make more sense to move this down to the flash targets?
I also don't understand how this works using a tab at the beginning. That should make it part of above's recipe, no?

Reply:
ah my bad, yes better to have it close to the flash targets